sdk/opendp/whitenoise/synthesizers/mwem/mwem.py

```diff
@@ -6,9 +7,7 @@
 import numpy as np
 import pandas as pd
 
-from sdgym.synthesizers.base import BaseSynthesizer
-
-class MWEMSynthesizer(BaseSynthesizer):

```

Comment:
Can we have an SDGYMBaseSynthesizer class that defines the minimal spec in synthesizers/base.py. 

It can be an abstract class to more clearly guide developers on what functions are needed for synthetic data generation.

If SDGYM were 1.0 we would depend on it directly, for now we can start off work like this. And then reach out to see if the collaboration can be more close.

Reply:
Gotcha, added